src/build.config.js

```diff
@@ -62,6 +62,14 @@ export default {
         icon: 'thorchain.svg',
         type: SwapProviderType.THORCHAIN,
         thornode: 'https://testnet.thornode.thorchain.info'
+      },
+      sovryn: {
+        name: 'Sovyrn',
+        icon: 'sovryn.svg',
+        type: SwapProviderType.SOVRYN,
+        routerAddress: '0x61172B53423E205a399640e5283e51FE60EC2256',
+        routerAddressRBTC: '0x2C468f9c82C20c37cd1606Cf3a09702f94910691',
+        rpcURL: 'https://testnet.sovryn.app/rpc'// 'https://public-node.testnet.rsk.co/'
```

Comment:
maybe we should remove the commented code 
// 'https://public-node.testnet.rsk.co/'